aptoide-installer/src/main/java/cm/aptoide/pt/installer/AptoideInstallPackageInfoMapper.kt

```diff
@@ -62,6 +63,32 @@ class AptoideInstallPackageInfoMapper @Inject constructor(
             )
           }
         }
+        appMeta.aab?.run {
+          val splits = this.splits.toMutableList()
+          val dynamicSplits = dynamicSplitsUseCase.getDynamicSplits(app.md5)
+
+          dynamicSplits.forEach {
+            if (it.deliveryTypes.contains("INSTALL_TIME")) {
```

Comment:
We need to map all splits, assigning the corresponding type for of them.
You can directly map `deliveryTypes` from the BE answer to the type enum.